apps/src/javalab/JavalabDialog.jsx

```diff
@@ -45,6 +50,12 @@ export default class JavalabDialog extends Component {
         >
           <div style={styles.message}>{message}</div>
           <div style={styles.buttons}>
+            {showSpinner && (
+              <i
+                className={classNames('fa', 'fa-spin', 'fa-spinner')}
```

Comment:
possible hot-take: if an element's classes are a static string, i don't know that we need to use the classnames lib -- we could "hard-code" the classes until we have conditional classes (`className="fa fa-spin fa-spinner"`)

Reply:
I think you're right, I might have originally had this because I thought I would need another classname here but ended up not needing one